.lintr

```diff
@@ -1,5 +1,6 @@
 linters: linters_with_defaults(
   line_length_linter = line_length_linter(120),
   cyclocomp_linter = NULL,
-  object_usage_linter = NULL
+  object_usage_linter = NULL,
+  indentation_linter = NULL
```

Comment:
why disable this?

Reply:
The code was restyled automatically by a GitHub auto-action when code is changed in accordance with indentation warnings by `linter`. This issue of indentation warnings was encountered across most of our packages, that's why  `indentation_linter ` is set to `NULL.`

warnings:
<img width="535" alt="image" src="https://github.com/insightsengineering/teal.slice/assets/6700955/97f545f5-c8f6-4b97-83d6-75d75ba2dd3a">

Reply:
These `indentation_linter` are most likely due to a conflict on lint and styling on `if` clauses

I've been using the (accepted) style below to get around the conflict. I found this issue a while ago in lintr repo (with a known face commenting there :smile:) https://github.com/r-lib/lintr/issues/2034

I can make the changes on a PR targetting this branch to solve this and avoid having the rule disabled

````r
     # lint error
      } else if (setequal(private$get_selected(), private$get_choices()) &&
        !anyNA(private$get_selected(), private$get_choices())) {
        TRUE
      } 
     # No lint error and styler doesn't reframe this
      } else if (
        setequal(private$get_selected(), private$get_choices()) &&
          !anyNA(private$get_selected(), private$get_choices())
      ) {
        TRUE
      }
````

Reply:
Thank you, @averissimo, for posting it here. that would be grate if the code can be formatted in a way that both {lintr} and {styler} accept, and looking forward to your PR for this. In my opinion, this should be handled as a separate issue since it affects all packages.  so, we can keep track of all the repositories  changes . What do you think?

>  I think it would be nice to report it somewhere (probably to lintr folks as a false positive) adding a minimal reprex based on the cases reported. I think this is the most common setup that should work.

@pawelru   should we duplicate https://github.com/r-lib/lintr/issues/2034 issue to keep track of it in `Coredev`

Reply:
I did a quick PR that targets this branch, but it can easily target `main` instead

Reply:
> should we duplicate [r-lib/lintr#2034](https://github.com/r-lib/lintr/issues/2034) issue to keep track of it in `Coredev`

IDK. Probably not. But definitely worth to add this as a code-comment what's the reason behind a given exclude